Cargo.toml

```diff
@@ -8,18 +8,17 @@ repository = "https://github.com/onomyprotocol/web30.git"
 version = "0.18.3"
 
 [dependencies]
-clarity = "0.5"
-futures = "0.3"
-hyper = {version = "0.14", features = ["full"]}
+clarity = { git = "https://github.com/AaronKutch/clarity.git", rev = "c50d7dca265d0b29da2326cae1b2996dc27b5ee1" }
```

Comment:
Don't forget to replace all links to your repo once we review and approve all the PRs.

Reply:
I think we have to do it one at a time, because using `/onomyprotocol/` doesn't work until the dependency PR is accepted

Reply:
I did allow maintainer changes, so if you want to merge more than one a day (we are 12 hours out of timezone sync) that's all you have to change